zad5/config/routes.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+Rails.application.routes.draw do
+  get 'artists/index'
+  get 'artists/destroy'
+  get 'artists/create'
+  get 'artists/update'
```

Comment:
te linijki są zbędne ponieważ `resources`, który jest w linii 14 tworzy już 7 ścieżek CRUDowych:
https://guides.rubyonrails.org/routing.html#crud-verbs-and-actions